### ISSUE TITLE
Any plan of updating to Mamba2?

### ISSUE THREAD

**Issue by csguoh** [MzeroMiko/VMamba#224]
Hi, 

Thanks for contributing to this awesome work.  As for as I know, the mamba2 has recently been proposed and shown promising performance against the previous version. I am curious that whether the authors will consider to update the Vmamba to allow a more powerful Mamba2?

Thx;D

**Comment by MzeroMiko** [comment]
Yes. We'll write the code soon. But it may take a while for us to train the model, or, maybe the new model will not be trained.

**Comment by csguoh** [comment]
An awesome open-source work. Looking forward to the update!
